Source/Core/Core/PowerPC/JitArm64/JitArm64_Branch.cpp

```diff
@@ -360,14 +314,14 @@ void JitArm64::bclrx(UGeckoInstruction inst)
 
   auto WA = gpr.GetScopedReg();
   Arm64GPRCache::ScopedARM64Reg WB;
-  if (conditional || inst.LK || IsDebuggingEnabled())
+  if (conditional || inst.LK || IsBranchWatchEnabled())
```

Comment:
```suggestion
  if (conditional || inst.LK)
```

Branch watch no longer uses WB.